migrate.go

```diff
@@ -0,0 +1,52 @@
+package sqlite
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+
+	"github.com/jmoiron/sqlx"
+)
+
+// Migrations is a list of migrations to run.
+type Migration func(ctx context.Context, db *sqlx.DB) error
+
+// Migrate run the migrations from the list that have not been executed.
```

Comment:
Sin the, corrigiendo el tiempo verbal y suena raro sin "yet" que eso no sé si es ortografía o solamente como suena.

```suggestion
// Migrate runs migrations from the list that have not been yet executed.
```